### ISSUE TITLE
Improve the accessibility of Hub and Desktop

### ISSUE THREAD

**Issue by rgardziel** [docker/roadmap#299]
Dockers GUI's are currently accessible for users who perceive colors in a variety of ways but fall short for other means of use. The experience using the product while limited to keyboard navigation or screenreaders is incomplete and doesn't meet AA compliance in all areas as outlined by ADA, Section 508, WCAG, IS5568, and more.

Which service(s) is this request for?
Docker Desktop and Docker Hub

**Comment by steverep** [comment]
I've been using Docker Desktop on Windows with a screen reader for years.  I've seen minimal to zero improvement in accessibility since I began using it.  When is this actually going to be worked?  And for the record, a11y isn't something you just put on a roadmap to improve and then close the ticket - it has to be part of the development process.
